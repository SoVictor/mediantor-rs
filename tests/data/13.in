121
1 870010529
1 -655028015
1 47384160
1 943908204
1 -778069009
1 20091569
1 584556596
1 34221871
1 -895177291
1 555522703
1 801683908
1 12953445
1 -88225141
1 438361945
1 11602390
1 -317702105
1 -639817748
1 -45240243
1 320090231
1 945087733
1 -158824385
1 801317839
1 -315516782
1 -319929919
1 382890602
1 -255606793
1 -848749161
1 -68452176
1 -449386413
1 -986812794
1 -931945206
1 -844746718
1 -127985255
1 53231935
1 -693877480
1 -946489062
1 -59632202
1 671366890
1 -575945144
1 -506296155
1 107053444
1 932931689
1 188651984
1 132384146
1 831661238
1 -512424500
1 181345911
1 -96924818
1 83177186
1 -470543958
1 662704645
1 -973248253
1 928282595
1 992879799
1 423384620
1 578770804
1 336474239
1 -500161759
1 453158311
1 335289958
1 423354100
1 738044642
0
1 206995772
0
1 -978190937
0
1 -446551651
0
1 -507703339
0
1 -665327874
0
1 944463574
0
1 212308471
0
1 -484016072
0
1 -296451008
0
1 127047205
0
1 781421977
0
1 -419178530
0
1 878589099
0
1 -981429781
0
1 -744732309
0
1 -232716080
0
1 795395734
0
1 -88134804
0
1 512560968
0
1 611468367
0
1 -733690719
0
1 -496578805
0
1 221665205
0
1 -916286250
0
1 658540378
0
1 -722883498
0
1 464411449
0
1 -570100198
0
1 27200893
0
