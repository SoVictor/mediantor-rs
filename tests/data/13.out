20091569
34221871
12953445
11602390
-45240243
-59632202
47384160
53231935
-68452176
-88225141
83177186
107053444
-96924818
127047205
-127985255
-158824385
-232716080
132384146
-88134804
181345911
188651984
-255606793
-296451008
206995772
-315516782
212308471
-317702105
221665205
-319929919
27200893
